e2e/crm7/pages/provider/case-details.js

```diff
@@ -6,8 +6,20 @@ export default class CasesDetailsPage {
     }
 
     async fillCaseDetails() {
+        await this.page.getByLabel('Main offence').waitFor({
+            state: 'visible',
+            timeout: 5000
```

Comment:
The default expect timeout should already be 5s

Reply:
added it default value if we wanted to change it in near future. I had issues with this element multiple times while testing.